external/README.md

```diff
@@ -226,7 +226,7 @@
 - Version: 1.2.11
 - License: zlib
 
-## iconv (libiconv)
+## libiconv (libiconv)
```

Comment:
Does the (libiconv) could be removed?

Reply:
Please add the whole request. I will look later for correct wording.
Thanks.